services/thermal_mgr/thermal_mgr.h

```diff
@@ -3,9 +3,11 @@
 #include "lm75bd.h"
 #include "errors.h"
 
+#define TEMPERATURE_WAIT_TIME pdMS_TO_TICKS(10)
+
 typedef enum {
-  THERMAL_MGR_EVENT_MEASURE_TEMP_CMD,
-  
+  THERMAL_MGR_EVENT_MEASURE_TEMP_CMD = 0,
+  THERMAL_OS_HANDLE = 1,
```

Comment:
Add the `THERMAL_MGR_EVENT_` prefix. It's important to give anything defined in a header file a name that prevents naming conflicts with other modules (i.e. try to make the name unique). In this case, we do this using the prefix.